src/imitation/testing/envs.py

```diff
@@ -1,35 +0,0 @@
-"""Helper methods for tests of custom Gym environments.
-
-This is used in the imitation test suite and may also be useful for users of this
-library.
-"""
-
-import gym
-
-
-def test_model_based(env: gym.Env) -> None:
```

Comment:
Do we have an analogous test in `seals`? If not, perhaps worth adding? It's a rather simple test admittedly but is testing some non-trivial invariants.

Reply:
Added an issue for this.